.github/workflows/build.yml

```
@@ -164,6 +164,37 @@ jobs:
           pushd android 1> /dev/null
           ./gradlew clean build check test
         working-directory: example
+  android-test:
+    name: "Android: instrumented tests"
+    strategy:
+      matrix:
+        api-level: [21, 29]
+    runs-on: macos-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Set up Node.js
+      uses: actions/setup-node@v1
+      with:
+        # Node has a bug where it crashes compiling a regular
+        # expression of react-native cli. Using a specific
+        # Node version helps to workaround this problem:
+        # https://github.com/facebook/react-native/issues/26598
+        node-version: 12.9.1
+    - uses: actions/checkout@v2
+    - name: Validate Gradle wrapper
+      uses: gradle/wrapper-validation-action@v1
+    - name: Install
+      run: yarn
```

We should use the node_modules cache:
```suggestion
      - name: Cache /node_modules
        uses: actions/cache@v2
        with:
          path: node_modules
          key: project-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
      - name: Install
        run: yarn
```